doc/man7/EVP_MD-MDC2.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-MDC2 - The MDC2 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing MDC2 digests through the B<EVP_MD> API.
+
+=head2 Identity
```

Comment:
Most of the time you have been using the plural form.

Reply:
Yes. But if there is only one, I figured singular is more appropriate

Reply:
I found a couple of occurrences where I got the distinction wrong.  Fixed.

Reply:
> Yes. But if there is only one, I figured singular is more appropriate

Well, if you prefer. But this means that whoever adds the next identity needs to modify the section name. Also, it is a common section, and giving it a common name would be more grep-w friendly. (For example, we use `=head1 NOTES` almost throughout, with seven exceptions, which I guess includes many locations with only one note.)